{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "c86cc83e",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "d37392e3",
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option(\"display.max_colwidth\",10000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "f783105a",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv(\"test.dat\",header=None,encoding='utf-8',sep=\"/r/n\",engine='python')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "id": "30527478",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ID   001R_FRG3G              Reviewed;         256 AA.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>AC   Q6GZX4;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DT   28-JUN-2011, integrated into UniProtKB/Swiss-Prot.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DT   19-JUL-2004, sequence version 1.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>DT   02-JUN-2021, entry version 42.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>DE   RecName: Full=Putative transcription factor 001R;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>GN   ORFNames=FV3-001R;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>OS   Frog virus 3 (isolate Goorha) (FV-3).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>OC   Viruses; Varidnaviria; Bamfordvirae; Nucleocytoviricota; Megaviricetes;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>OC   Pimascovirales; Iridoviridae; Alphairidovirinae; Ranavirus.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>OX   NCBI_TaxID=654924;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>OH   NCBI_TaxID=30343; Dryophytes versicolor (chameleon treefrog).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>OH   NCBI_TaxID=8404; Lithobates pipiens (Northern leopard frog) (Rana pipiens).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>OH   NCBI_TaxID=45438; Lithobates sylvaticus (Wood frog) (Rana sylvatica).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>OH   NCBI_TaxID=8316; Notophthalmus viridescens (Eastern newt) (Triturus viridescens).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>RN   [1]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>RP   NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA].</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>RX   PubMed=15165820; DOI=10.1016/j.virol.2004.02.019;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>RA   Tan W.G., Barkman T.J., Gregory Chinchar V., Essani K.;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>RT   \"Comparative genomic analyses of frog virus 3, type species of the genus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>RT   Ranavirus (family Iridoviridae).\";</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>RL   Virology 323:70-84(2004).</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>CC   -!- FUNCTION: Transcription activation. {ECO:0000305}.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>CC   ---------------------------------------------------------------------------</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/terms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>CC   Distributed under the Creative Commons Attribution (CC BY 4.0) License</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>CC   ---------------------------------------------------------------------------</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>DR   EMBL; AY548484; AAT09660.1; -; Genomic_DNA.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>DR   RefSeq; YP_031579.1; NC_005946.1.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>DR   SwissPalm; Q6GZX4; -.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>DR   GeneID; 2947773; -.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>DR   KEGG; vg:2947773; -.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>DR   Proteomes; UP000008770; Genome.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>DR   GO; GO:0046782; P:regulation of viral transcription; IEA:InterPro.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>DR   InterPro; IPR007031; Poxvirus_VLTF3.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>DR   Pfam; PF04947; Pox_VLTF3; 1.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>PE   4: Predicted;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>KW   Activator; Reference proteome; Transcription; Transcription regulation.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>FT   CHAIN           1..256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>FT                   /note=\"Putative transcription factor 001R\"</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>FT                   /id=\"PRO_0000410512\"</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>SQ   SEQUENCE   256 AA;  29735 MW;  B4840739BF7D4121 CRC64;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>MAFSAEDVLK EYDRRRRMEA LLLSLYYPND RKLLDYKEWS PPRVQVECPK APVEWNNPPS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>EKGLIVGHFS GIKYKGEKAQ ASEVDVNKMC CWVSKFKDAM RRYQGIQTCK IPGKVLSDLD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>AKIKAYNLTV EGVEGFVRYS RVTKQHVAAF LKELRHSKQY ENVNLIHYIL TDKRVDIQHL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>EKDLVKDFKA LVESAHRMRQ GHMINVKYIL YQLLKKHGHG PDGPDILTVK TGSKGVLYDD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>SFRKIYTDLG WKFTPL</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                         0\n",
       "0                                   ID   001R_FRG3G              Reviewed;         256 AA.\n",
       "1                                                                             AC   Q6GZX4;\n",
       "2                                  DT   28-JUN-2011, integrated into UniProtKB/Swiss-Prot.\n",
       "3                                                    DT   19-JUL-2004, sequence version 1.\n",
       "4                                                      DT   02-JUN-2021, entry version 42.\n",
       "5                                   DE   RecName: Full=Putative transcription factor 001R;\n",
       "6                                                                  GN   ORFNames=FV3-001R;\n",
       "7                                               OS   Frog virus 3 (isolate Goorha) (FV-3).\n",
       "8             OC   Viruses; Varidnaviria; Bamfordvirae; Nucleocytoviricota; Megaviricetes;\n",
       "9                         OC   Pimascovirales; Iridoviridae; Alphairidovirinae; Ranavirus.\n",
       "10                                                                 OX   NCBI_TaxID=654924;\n",
       "11                      OH   NCBI_TaxID=30343; Dryophytes versicolor (chameleon treefrog).\n",
       "12        OH   NCBI_TaxID=8404; Lithobates pipiens (Northern leopard frog) (Rana pipiens).\n",
       "13              OH   NCBI_TaxID=45438; Lithobates sylvaticus (Wood frog) (Rana sylvatica).\n",
       "14  OH   NCBI_TaxID=8316; Notophthalmus viridescens (Eastern newt) (Triturus viridescens).\n",
       "15                                                                                RN   [1]\n",
       "16                                     RP   NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA].\n",
       "17                                  RX   PubMed=15165820; DOI=10.1016/j.virol.2004.02.019;\n",
       "18                            RA   Tan W.G., Barkman T.J., Gregory Chinchar V., Essani K.;\n",
       "19           RT   \"Comparative genomic analyses of frog virus 3, type species of the genus\n",
       "20                                                 RT   Ranavirus (family Iridoviridae).\";\n",
       "21                                                          RL   Virology 323:70-84(2004).\n",
       "22                             CC   -!- FUNCTION: Transcription activation. {ECO:0000305}.\n",
       "23        CC   ---------------------------------------------------------------------------\n",
       "24           CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/terms\n",
       "25             CC   Distributed under the Creative Commons Attribution (CC BY 4.0) License\n",
       "26        CC   ---------------------------------------------------------------------------\n",
       "27                                        DR   EMBL; AY548484; AAT09660.1; -; Genomic_DNA.\n",
       "28                                                  DR   RefSeq; YP_031579.1; NC_005946.1.\n",
       "29                                                              DR   SwissPalm; Q6GZX4; -.\n",
       "30                                                                DR   GeneID; 2947773; -.\n",
       "31                                                               DR   KEGG; vg:2947773; -.\n",
       "32                                                    DR   Proteomes; UP000008770; Genome.\n",
       "33                 DR   GO; GO:0046782; P:regulation of viral transcription; IEA:InterPro.\n",
       "34                                               DR   InterPro; IPR007031; Poxvirus_VLTF3.\n",
       "35                                                       DR   Pfam; PF04947; Pox_VLTF3; 1.\n",
       "36                                                                      PE   4: Predicted;\n",
       "37            KW   Activator; Reference proteome; Transcription; Transcription regulation.\n",
       "38                                                             FT   CHAIN           1..256\n",
       "39                         FT                   /note=\"Putative transcription factor 001R\"\n",
       "40                                               FT                   /id=\"PRO_0000410512\"\n",
       "41                             SQ   SEQUENCE   256 AA;  29735 MW;  B4840739BF7D4121 CRC64;\n",
       "42                       MAFSAEDVLK EYDRRRRMEA LLLSLYYPND RKLLDYKEWS PPRVQVECPK APVEWNNPPS\n",
       "43                       EKGLIVGHFS GIKYKGEKAQ ASEVDVNKMC CWVSKFKDAM RRYQGIQTCK IPGKVLSDLD\n",
       "44                       AKIKAYNLTV EGVEGFVRYS RVTKQHVAAF LKELRHSKQY ENVNLIHYIL TDKRVDIQHL\n",
       "45                       EKDLVKDFKA LVESAHRMRQ GHMINVKYIL YQLLKKHGHG PDGPDILTVK TGSKGVLYDD\n",
       "46                                                                       SFRKIYTDLG WKFTPL"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head(200)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "e183e52b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0    FT                   /note=\"Putative transcription factor 001R\"\n",
       "Name: 39, dtype: object"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.iloc[39]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "id": "90ba3c7f",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_ft(info):\n",
    "    if str(info).find(\"FT \") != -1:\n",
    "        info_list = str(info).split(\"FT\")\n",
    "        print(\"info_list\",info_list)\n",
    "    return info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "id": "8b70d12f",
   "metadata": {},
   "outputs": [],
   "source": [
    "def process(info):\n",
    "    get_ft(info)\n",
    "    return info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "9116f0ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "info_list ['', '   CHAIN           1..256']\n",
      "info_list ['', '                   /note=\"Putative transcription factor 001R\"']\n",
      "info_list ['', '                   /id=\"PRO_0000410512\"']\n"
     ]
    }
   ],
   "source": [
    "df[1] = df[0].apply(lambda info:process(info))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "563a923b",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "id": "94dfacdc",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "id": "af913303",
   "metadata": {},
   "outputs": [],
   "source": [
    "overview_path = os.path.join(r\"C:\\Users\\chen\\Desktop\\data\\overview\")\n",
    "phenotypes_path = os.path.join(r\"C:\\Users\\chen\\Desktop\\data\\phenotypes\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "id": "4a6f468f",
   "metadata": {},
   "outputs": [],
   "source": [
    "home_df = pd.read_excel(r\"C:\\Users\\chen\\Desktop\\data\\home.xlsx\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "9e15f72b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Syndrome Name</th>\n",
       "      <th>Location</th>\n",
       "      <th>Genotype / Class</th>\n",
       "      <th>Size</th>\n",
       "      <th>Grade</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>12q14 microdeletion syndrome</td>\n",
       "      <td>12\\n:\\n64678139\\n-\\n68251745\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>3.57 Mb</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>15q13.3 microdeletion syndrome</td>\n",
       "      <td>15\\n:\\n30618103\\n-\\n32153206\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>1.54 Mb</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>15q24 recurrent microdeletion syndrome</td>\n",
       "      <td>15\\n:\\n74120302\\n-\\n75680570\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>1.56 Mb</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            Syndrome Name                              Location        Genotype / Class     Size  Grade\n",
       "0                                    None                                  None                    None     None    NaN\n",
       "1                                    None                                  None                    None     None    NaN\n",
       "2            12q14 microdeletion syndrome  12\\n:\\n64678139\\n-\\n68251745\\nGRCh38  Heterozygous\\nDeletion  3.57 Mb    NaN\n",
       "3          15q13.3 microdeletion syndrome  15\\n:\\n30618103\\n-\\n32153206\\nGRCh38  Heterozygous\\nDeletion  1.54 Mb    NaN\n",
       "4  15q24 recurrent microdeletion syndrome  15\\n:\\n74120302\\n-\\n75680570\\nGRCh38  Heterozygous\\nDeletion  1.56 Mb    NaN"
      ]
     },
     "execution_count": 112,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "home_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "fb1a310e",
   "metadata": {},
   "outputs": [],
   "source": [
    "home_df = home_df.where(home_df.notnull(),None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "53eb22ac",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "07f51f54",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "id": "3b02d47b",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_overview(info):\n",
    "    if not info or info == np.nan:\n",
    "        return\n",
    "    try:\n",
    "        path = os.path.join(overview_path,f\"\" + str(info).replace(\"/\",\"\") + \".csv\")\n",
    "        overview_df = pd.read_csv(path)\n",
    "        if overview_df.empty:\n",
    "            return\n",
    "    #     display(overview_df)\n",
    "        overview_text = overview_df.iloc[0][\"overview\"]\n",
    "    except Exception as e:\n",
    "        print(e.__str__())\n",
    "        return\n",
    "    return overview_text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "id": "035db4d8",
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "home_df[\"overview-Clinical\"] = home_df[\"Syndrome Name\"].apply(lambda x:get_overview(x))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "id": "0648caa9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Syndrome Name</th>\n",
       "      <th>Location</th>\n",
       "      <th>Genotype / Class</th>\n",
       "      <th>Size</th>\n",
       "      <th>Grade</th>\n",
       "      <th>overview-Clinical</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>12q14 microdeletion syndrome</td>\n",
       "      <td>12\\n:\\n64678139\\n-\\n68251745\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>3.57 Mb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: The three children reported by Menten et al all had a similar and characteristic phenotype consisting of mild mental retardation, failure to thrive in infancy, proportionate short stature and osteopoikilosis. The common 3.44Mb deleted region contains the LEMD3 gene. Loss of function mutations in this gene cause osteopoikilosis and the Bushke-Ollendorff syndrome (BOS). LEMD3 mutations have also been identified in melorheostosis patients who belong to a family with BOS but they are rarely present in sporadic melorheostosis\\nFailure to thrive, short stature and mental retardation are not observed in patients with osteopoikilosis or melorheostosis and these features are presumed to be attributable to another dosage sensitive gene(s) in the common deleted region.\\nSize of deletion: Variable deletion size, varying between 3.44 and 6 Mb in size with a 3.44 Mb common deleted region.\\nOrigin of deletion: The deletion occurred as a de novo event. The breakpoints are variable and apparently non-recurrent. Given the apparent absence of low copy repeats\\nnear the 12q breakpoints described here, a mechanism of non-homologous end-joining (NHEJ) may be responsible for the occurrence of the microdeletions in the three published cases\\nExpert advisor: Geert Mortier, Department of Medical Genetics, Ghent University Hospital, De Pintelaan 185, B-9000 Ghent, Belgium\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.orpha.net\\nrarediseases.info.nih.gov</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>15q13.3 microdeletion syndrome</td>\n",
       "      <td>15\\n:\\n30618103\\n-\\n32153206\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>1.54 Mb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: This microdeletion syndrome was first described by Sharp et al in 2008 and is characterised by developmental delay with mild/moderated learning disability, seizures and subtle facial dysmorphism. A high percentage of patients have minor hand anomalies eg. short 4th metacarpal or 5th finger clinodactyly. Expressivity of the condition appears to be quite variable. There is no striking facial gestalt and given the lack of distinctive features in the phenotype, this is unlikely to become a disorder which is easily recognisable clinically.\\nThe seizure disorder is variable; 7 of the 9 patients in Sharp's series had seizures or an abnormal EEG. The seizures varied from absence seizures to generalised tonic-clonic seizures to myoclonic epilepsy. One patient had intractable epilepsy. The deletion encompasses CHRNA7(cholinergic receptor, neuronal nicotinic, alpha polypeptide 7), a synaptic ion channel protein that mediates neuronal signal transmission.\\nSharp et al estimate an approximate population incidence of ~1/40,000.\\nSize of deletion: There is a shared 1.5 Mb region between segmental duplications at BP4 and BP5 containing six known genes. Although they occur more rarely, there is also a larger recurrent deletion between BP3 and BP5 associated with this phenotype\\nOrigin of deletion: This region of chromosome 15q containes a complex set of segmental duplications, termed duplication blocks (eg. BP4). The BP4 and BP5 are in inverted orientation in a proportion of individuals in the general population which facilitates non-homologous recombination of this region at meiosis.\\nExpert advisor: Dr Evan Eichler, Department of Genome Sciences, University of Washington and Howard Hughes Medical Institute, Seattle, Washington, USA\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>15q24 recurrent microdeletion syndrome</td>\n",
       "      <td>15\\n:\\n74120302\\n-\\n75680570\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>1.56 Mb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: 15q24 microdeletion syndrome is a newly defined syndrome (2007). Of the four patients described to date (Sharp) all had mild mental retardation (IQ 60-70), together with growth retardation, microcephaly, digital abnormalities, hypospadias (3/4) and loose connective tissue. A degree of facial resemblence was noted with high anterior hair line, broad medial eyebrows, hypertelorism, downslanted palpebral fissures, broad nasal base, long smooth philtrum and full lower lip.\\nSize of deletion: 1-7-3.9Mb. In Sharp's series, high-resolution analysis showed that in three patients both proximal and distal breakpoints co-localized to highly identical segmental duplications, suggesting non-allelic homologous recombination (NAHR) as the likely mechanism of origin.However,in a fourth case, which was clinically indistinguishable, both proximal and distal breakpoints were located in unique sequence.\\nOrigin of deletion: The majority of cases observed to date appear due to de novo deletions.\\nExpert advisor: Dr Andy Sharp, University of Washington, Dept. of Genome Sciences, Seattle, USA\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>Wolf-Hirschhorn Syndrome</td>\n",
       "      <td>4\\n:\\n1567470\\n-\\n2108509\\nGRCh38</td>\n",
       "      <td>Heterozygous\\nDeletion</td>\n",
       "      <td>541.04 kb</td>\n",
       "      <td>1.0</td>\n",
       "      <td>Clinical - Characterised by low birth-weight and post-natal failure to thrive, microcephaly, developmental delay and hypotonia. There is a characteristic facial appearance with sagging everted lower eyelids, a 'Greek-helmet' profile, a short nose and very short philtrum. They may have iris colobomas. Seizures are common. Some have a visible deletion with varying breakpoints on 4p, others have a cryptic deletion requiring FISH to make the diagnosis. From Shannon's study, a minimum birth incidence of 1 in 95 896 was calculated. They found that the crude infant mortality rate was 17% (23/132) and in the first two years of life the mortality rate was 21% (28/132). Cases with large de novo deletions (proximal to and including 4p15.2) were more likely to have died than those with smaller deletions.\\nSize of deletion - Variable, genotype-phenotype correlations in WHS mostly depend on the size of the deletion, a deletion of &gt;3.5 Mb resulting in a mild phenotype, in which malformations are absent (Zollino).\\nOrigin of deletion -\\nExpert adviser:\\nLinks to support groups:\\nwolfhirschhorn.org\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>Xp11.22-linked intellectual disability</td>\n",
       "      <td>X\\n:\\n53374149\\n-\\n53656331\\nGRCh38</td>\n",
       "      <td>Unknown\\nDuplication</td>\n",
       "      <td>282.18 kb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: Data Awaited\\nLinks to further information:\\nwww.geneclinics.org\\nomim.org</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>Xp11.22-p11.23 Microduplication</td>\n",
       "      <td>X\\n:\\n48476161\\n-\\n52374518\\nGRCh38</td>\n",
       "      <td>Unknown\\nDuplication</td>\n",
       "      <td>3.90 Mb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: Data Awaited\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.orpha.net\\nomim.org</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>Xq28 (MECP2) duplication</td>\n",
       "      <td>X\\n:\\n154021812\\n-\\n154097731\\nGRCh38</td>\n",
       "      <td>Unknown\\nDuplication</td>\n",
       "      <td>75.92 kb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: Severe mental retardation, severe hypotonia with onset at birth and in the majority of patients a progressive lower-limb spasticity. Absent or very limited speech. Seizures are common as are recurrent severe infections. Growth parameters (including OFC) fall within the normal range. Carrier females showed complete skewing of X-inactivation. By analysis of patients with duplications in this region, Van Esch concluded that increased gene dosage of MECP2 was the most likely explanation for the severe MR phenotype present.\\nSize of duplication: Van Esch found variable duplication size, between 0.4-0.8Mb including the LICAM and MECP2 genes, but no consistent centromeric or telomeric breakpoint.\\nOrigin of duplication: Not known.\\nExpert adviser:Dr Hilde Van Esch, Centre for Human genetics, University Hosptial Gasthuisberg, Leuven, Belgium\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>Xq28 Microduplication</td>\n",
       "      <td>X\\n:\\n154396222\\n-\\n154653579\\nGRCh38</td>\n",
       "      <td>Unknown\\nDuplication</td>\n",
       "      <td>257.36 kb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Clinical: Data Awaited\\nLinks to further information:\\nwww.orpha.net</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>68 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             Syndrome Name                               Location        Genotype / Class       Size  Grade  \\\n",
       "0                                     None                                   None                    None       None    NaN   \n",
       "1                                     None                                   None                    None       None    NaN   \n",
       "2             12q14 microdeletion syndrome   12\\n:\\n64678139\\n-\\n68251745\\nGRCh38  Heterozygous\\nDeletion    3.57 Mb    NaN   \n",
       "3           15q13.3 microdeletion syndrome   15\\n:\\n30618103\\n-\\n32153206\\nGRCh38  Heterozygous\\nDeletion    1.54 Mb    NaN   \n",
       "4   15q24 recurrent microdeletion syndrome   15\\n:\\n74120302\\n-\\n75680570\\nGRCh38  Heterozygous\\nDeletion    1.56 Mb    NaN   \n",
       "..                                     ...                                    ...                     ...        ...    ...   \n",
       "63                Wolf-Hirschhorn Syndrome      4\\n:\\n1567470\\n-\\n2108509\\nGRCh38  Heterozygous\\nDeletion  541.04 kb    1.0   \n",
       "64  Xp11.22-linked intellectual disability    X\\n:\\n53374149\\n-\\n53656331\\nGRCh38    Unknown\\nDuplication  282.18 kb    NaN   \n",
       "65         Xp11.22-p11.23 Microduplication    X\\n:\\n48476161\\n-\\n52374518\\nGRCh38    Unknown\\nDuplication    3.90 Mb    NaN   \n",
       "66                Xq28 (MECP2) duplication  X\\n:\\n154021812\\n-\\n154097731\\nGRCh38    Unknown\\nDuplication   75.92 kb    NaN   \n",
       "67                   Xq28 Microduplication  X\\n:\\n154396222\\n-\\n154653579\\nGRCh38    Unknown\\nDuplication  257.36 kb    NaN   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 overview-Clinical  \n",
       "0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             None  \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             None  \n",
       "2                                                                                                                                                                                                                                                                                                                                                                                         Clinical: The three children reported by Menten et al all had a similar and characteristic phenotype consisting of mild mental retardation, failure to thrive in infancy, proportionate short stature and osteopoikilosis. The common 3.44Mb deleted region contains the LEMD3 gene. Loss of function mutations in this gene cause osteopoikilosis and the Bushke-Ollendorff syndrome (BOS). LEMD3 mutations have also been identified in melorheostosis patients who belong to a family with BOS but they are rarely present in sporadic melorheostosis\\nFailure to thrive, short stature and mental retardation are not observed in patients with osteopoikilosis or melorheostosis and these features are presumed to be attributable to another dosage sensitive gene(s) in the common deleted region.\\nSize of deletion: Variable deletion size, varying between 3.44 and 6 Mb in size with a 3.44 Mb common deleted region.\\nOrigin of deletion: The deletion occurred as a de novo event. The breakpoints are variable and apparently non-recurrent. Given the apparent absence of low copy repeats\\nnear the 12q breakpoints described here, a mechanism of non-homologous end-joining (NHEJ) may be responsible for the occurrence of the microdeletions in the three published cases\\nExpert advisor: Geert Mortier, Department of Medical Genetics, Ghent University Hospital, De Pintelaan 185, B-9000 Ghent, Belgium\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.orpha.net\\nrarediseases.info.nih.gov  \n",
       "3   Clinical: This microdeletion syndrome was first described by Sharp et al in 2008 and is characterised by developmental delay with mild/moderated learning disability, seizures and subtle facial dysmorphism. A high percentage of patients have minor hand anomalies eg. short 4th metacarpal or 5th finger clinodactyly. Expressivity of the condition appears to be quite variable. There is no striking facial gestalt and given the lack of distinctive features in the phenotype, this is unlikely to become a disorder which is easily recognisable clinically.\\nThe seizure disorder is variable; 7 of the 9 patients in Sharp's series had seizures or an abnormal EEG. The seizures varied from absence seizures to generalised tonic-clonic seizures to myoclonic epilepsy. One patient had intractable epilepsy. The deletion encompasses CHRNA7(cholinergic receptor, neuronal nicotinic, alpha polypeptide 7), a synaptic ion channel protein that mediates neuronal signal transmission.\\nSharp et al estimate an approximate population incidence of ~1/40,000.\\nSize of deletion: There is a shared 1.5 Mb region between segmental duplications at BP4 and BP5 containing six known genes. Although they occur more rarely, there is also a larger recurrent deletion between BP3 and BP5 associated with this phenotype\\nOrigin of deletion: This region of chromosome 15q containes a complex set of segmental duplications, termed duplication blocks (eg. BP4). The BP4 and BP5 are in inverted orientation in a proportion of individuals in the general population which facilitates non-homologous recombination of this region at meiosis.\\nExpert advisor: Dr Evan Eichler, Department of Genome Sciences, University of Washington and Howard Hughes Medical Institute, Seattle, Washington, USA\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net  \n",
       "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Clinical: 15q24 microdeletion syndrome is a newly defined syndrome (2007). Of the four patients described to date (Sharp) all had mild mental retardation (IQ 60-70), together with growth retardation, microcephaly, digital abnormalities, hypospadias (3/4) and loose connective tissue. A degree of facial resemblence was noted with high anterior hair line, broad medial eyebrows, hypertelorism, downslanted palpebral fissures, broad nasal base, long smooth philtrum and full lower lip.\\nSize of deletion: 1-7-3.9Mb. In Sharp's series, high-resolution analysis showed that in three patients both proximal and distal breakpoints co-localized to highly identical segmental duplications, suggesting non-allelic homologous recombination (NAHR) as the likely mechanism of origin.However,in a fourth case, which was clinically indistinguishable, both proximal and distal breakpoints were located in unique sequence.\\nOrigin of deletion: The majority of cases observed to date appear due to de novo deletions.\\nExpert advisor: Dr Andy Sharp, University of Washington, Dept. of Genome Sciences, Seattle, USA\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net  \n",
       "..                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             ...  \n",
       "63                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Clinical - Characterised by low birth-weight and post-natal failure to thrive, microcephaly, developmental delay and hypotonia. There is a characteristic facial appearance with sagging everted lower eyelids, a 'Greek-helmet' profile, a short nose and very short philtrum. They may have iris colobomas. Seizures are common. Some have a visible deletion with varying breakpoints on 4p, others have a cryptic deletion requiring FISH to make the diagnosis. From Shannon's study, a minimum birth incidence of 1 in 95 896 was calculated. They found that the crude infant mortality rate was 17% (23/132) and in the first two years of life the mortality rate was 21% (28/132). Cases with large de novo deletions (proximal to and including 4p15.2) were more likely to have died than those with smaller deletions.\\nSize of deletion - Variable, genotype-phenotype correlations in WHS mostly depend on the size of the deletion, a deletion of >3.5 Mb resulting in a mild phenotype, in which malformations are absent (Zollino).\\nOrigin of deletion -\\nExpert adviser:\\nLinks to support groups:\\nwolfhirschhorn.org\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net  \n",
       "64                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Clinical: Data Awaited\\nLinks to further information:\\nwww.geneclinics.org\\nomim.org  \n",
       "65                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Clinical: Data Awaited\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.orpha.net\\nomim.org  \n",
       "66                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Clinical: Severe mental retardation, severe hypotonia with onset at birth and in the majority of patients a progressive lower-limb spasticity. Absent or very limited speech. Seizures are common as are recurrent severe infections. Growth parameters (including OFC) fall within the normal range. Carrier females showed complete skewing of X-inactivation. By analysis of patients with duplications in this region, Van Esch concluded that increased gene dosage of MECP2 was the most likely explanation for the severe MR phenotype present.\\nSize of duplication: Van Esch found variable duplication size, between 0.4-0.8Mb including the LICAM and MECP2 genes, but no consistent centromeric or telomeric breakpoint.\\nOrigin of duplication: Not known.\\nExpert adviser:Dr Hilde Van Esch, Centre for Human genetics, University Hosptial Gasthuisberg, Leuven, Belgium\\nLinks to support groups:\\nwww.rarechromo.org\\nLinks to further information:\\nwww.ncbi.nlm.nih.gov\\nwww.orpha.net  \n",
       "67                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Clinical: Data Awaited\\nLinks to further information:\\nwww.orpha.net  \n",
       "\n",
       "[68 rows x 6 columns]"
      ]
     },
     "execution_count": 144,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "home_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9691da0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "877d6013",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5f1d89b7",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d2e17387",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9a019f2a",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "2ce0524f",
   "metadata": {},
   "outputs": [],
   "source": [
    "df2 = df.T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "197d080d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>37</th>\n",
       "      <th>38</th>\n",
       "      <th>39</th>\n",
       "      <th>40</th>\n",
       "      <th>41</th>\n",
       "      <th>42</th>\n",
       "      <th>43</th>\n",
       "      <th>44</th>\n",
       "      <th>45</th>\n",
       "      <th>46</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ID</td>\n",
       "      <td>AC</td>\n",
       "      <td>DT</td>\n",
       "      <td>DT</td>\n",
       "      <td>DT</td>\n",
       "      <td>DE</td>\n",
       "      <td>GN</td>\n",
       "      <td>OS</td>\n",
       "      <td>OC</td>\n",
       "      <td>OC</td>\n",
       "      <td>...</td>\n",
       "      <td>KW</td>\n",
       "      <td>FT</td>\n",
       "      <td>FT</td>\n",
       "      <td>FT</td>\n",
       "      <td>SQ</td>\n",
       "      <td>MAFSAEDVLK EYDRRRRMEA LLLSLYYPND RKLLDYKEWS PP...</td>\n",
       "      <td>EKGLIVGHFS GIKYKGEKAQ ASEVDVNKMC CWVSKFKDAM RR...</td>\n",
       "      <td>AKIKAYNLTV EGVEGFVRYS RVTKQHVAAF LKELRHSKQY EN...</td>\n",
       "      <td>EKDLVKDFKA LVESAHRMRQ GHMINVKYIL YQLLKKHGHG PD...</td>\n",
       "      <td>SFRKIYTDLG WKFTPL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>001R_FRG3G</td>\n",
       "      <td>Q6GZX4;</td>\n",
       "      <td>28-JUN-2011, integrated into UniProtKB/Swiss-...</td>\n",
       "      <td>19-JUL-2004, sequence version 1.</td>\n",
       "      <td>02-JUN-2021, entry version 42.</td>\n",
       "      <td>RecName: Full=Putative transcription factor 0...</td>\n",
       "      <td>ORFNames=FV3-001R;</td>\n",
       "      <td>Frog virus 3 (isolate Goorha) (FV-3).</td>\n",
       "      <td>Viruses; Varidnaviria; Bamfordvirae; Nucleocy...</td>\n",
       "      <td>Pimascovirales; Iridoviridae; Alphairidovirin...</td>\n",
       "      <td>...</td>\n",
       "      <td>Activator; Reference proteome; Transcription;...</td>\n",
       "      <td>CHAIN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SEQUENCE</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>256 AA;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>29735 MW;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>B4840739BF7D4121 CRC64;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 47 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "            0         1                                                  2   \\\n",
       "0           ID        AC                                                 DT   \n",
       "1   001R_FRG3G   Q6GZX4;   28-JUN-2011, integrated into UniProtKB/Swiss-...   \n",
       "2          NaN      None                                               None   \n",
       "3          NaN      None                                               None   \n",
       "4          NaN      None                                               None   \n",
       "\n",
       "                                  3                                4   \\\n",
       "0                                 DT                               DT   \n",
       "1   19-JUL-2004, sequence version 1.   02-JUN-2021, entry version 42.   \n",
       "2                               None                             None   \n",
       "3                               None                             None   \n",
       "4                               None                             None   \n",
       "\n",
       "                                                  5                    6   \\\n",
       "0                                                 DE                   GN   \n",
       "1   RecName: Full=Putative transcription factor 0...   ORFNames=FV3-001R;   \n",
       "2                                               None                 None   \n",
       "3                                               None                 None   \n",
       "4                                               None                 None   \n",
       "\n",
       "                                       7   \\\n",
       "0                                      OS   \n",
       "1   Frog virus 3 (isolate Goorha) (FV-3).   \n",
       "2                                    None   \n",
       "3                                    None   \n",
       "4                                    None   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                                 OC   \n",
       "1   Viruses; Varidnaviria; Bamfordvirae; Nucleocy...   \n",
       "2                                               None   \n",
       "3                                               None   \n",
       "4                                               None   \n",
       "\n",
       "                                                  9   ...  \\\n",
       "0                                                 OC  ...   \n",
       "1   Pimascovirales; Iridoviridae; Alphairidovirin...  ...   \n",
       "2                                               None  ...   \n",
       "3                                               None  ...   \n",
       "4                                               None  ...   \n",
       "\n",
       "                                                  37      38   39   40  \\\n",
       "0                                                 KW      FT   FT   FT   \n",
       "1   Activator; Reference proteome; Transcription;...   CHAIN  NaN  NaN   \n",
       "2                                               None     NaN  NaN  NaN   \n",
       "3                                               None     NaN  NaN  NaN   \n",
       "4                                               None     NaN  NaN  NaN   \n",
       "\n",
       "                        41                                                 42  \\\n",
       "0                       SQ  MAFSAEDVLK EYDRRRRMEA LLLSLYYPND RKLLDYKEWS PP...   \n",
       "1                 SEQUENCE                                               None   \n",
       "2                  256 AA;                                               None   \n",
       "3                29735 MW;                                               None   \n",
       "4  B4840739BF7D4121 CRC64;                                               None   \n",
       "\n",
       "                                                  43  \\\n",
       "0  EKGLIVGHFS GIKYKGEKAQ ASEVDVNKMC CWVSKFKDAM RR...   \n",
       "1                                               None   \n",
       "2                                               None   \n",
       "3                                               None   \n",
       "4                                               None   \n",
       "\n",
       "                                                  44  \\\n",
       "0  AKIKAYNLTV EGVEGFVRYS RVTKQHVAAF LKELRHSKQY EN...   \n",
       "1                                               None   \n",
       "2                                               None   \n",
       "3                                               None   \n",
       "4                                               None   \n",
       "\n",
       "                                                  45                 46  \n",
       "0  EKDLVKDFKA LVESAHRMRQ GHMINVKYIL YQLLKKHGHG PD...  SFRKIYTDLG WKFTPL  \n",
       "1                                               None               None  \n",
       "2                                               None               None  \n",
       "3                                               None               None  \n",
       "4                                               None               None  \n",
       "\n",
       "[5 rows x 47 columns]"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df2.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "6d26c234",
   "metadata": {},
   "outputs": [],
   "source": [
    "df2.columns = df2.iloc[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "fd77dacd",
   "metadata": {},
   "outputs": [],
   "source": [
    "df2 = df2.drop(index=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "b1d910c2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>AC</th>\n",
       "      <th>DT</th>\n",
       "      <th>DT</th>\n",
       "      <th>DT</th>\n",
       "      <th>DE</th>\n",
       "      <th>GN</th>\n",
       "      <th>OS</th>\n",
       "      <th>OC</th>\n",
       "      <th>OC</th>\n",
       "      <th>...</th>\n",
       "      <th>KW</th>\n",
       "      <th>FT</th>\n",
       "      <th>FT</th>\n",
       "      <th>FT</th>\n",
       "      <th>SQ</th>\n",
       "      <th>MAFSAEDVLK EYDRRRRMEA LLLSLYYPND RKLLDYKEWS PPRVQVECPK APVEWNNPPS</th>\n",
       "      <th>EKGLIVGHFS GIKYKGEKAQ ASEVDVNKMC CWVSKFKDAM RRYQGIQTCK IPGKVLSDLD</th>\n",
       "      <th>AKIKAYNLTV EGVEGFVRYS RVTKQHVAAF LKELRHSKQY ENVNLIHYIL TDKRVDIQHL</th>\n",
       "      <th>EKDLVKDFKA LVESAHRMRQ GHMINVKYIL YQLLKKHGHG PDGPDILTVK TGSKGVLYDD</th>\n",
       "      <th>SFRKIYTDLG WKFTPL</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>001R_FRG3G</td>\n",
       "      <td>Q6GZX4;</td>\n",
       "      <td>28-JUN-2011, integrated into UniProtKB/Swiss-...</td>\n",
       "      <td>19-JUL-2004, sequence version 1.</td>\n",
       "      <td>02-JUN-2021, entry version 42.</td>\n",
       "      <td>RecName: Full=Putative transcription factor 0...</td>\n",
       "      <td>ORFNames=FV3-001R;</td>\n",
       "      <td>Frog virus 3 (isolate Goorha) (FV-3).</td>\n",
       "      <td>Viruses; Varidnaviria; Bamfordvirae; Nucleocy...</td>\n",
       "      <td>Pimascovirales; Iridoviridae; Alphairidovirin...</td>\n",
       "      <td>...</td>\n",
       "      <td>Activator; Reference proteome; Transcription;...</td>\n",
       "      <td>CHAIN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SEQUENCE</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>256 AA;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>29735 MW;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>B4840739BF7D4121 CRC64;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>1..256</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Reviewed;</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>/note=\"Putative transcription factor 001R\"</td>\n",
       "      <td>/id=\"PRO_0000410512\"</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>256 AA.</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>12 rows × 47 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "0            ID        AC                                                 DT  \\\n",
       "1    001R_FRG3G   Q6GZX4;   28-JUN-2011, integrated into UniProtKB/Swiss-...   \n",
       "2           NaN      None                                               None   \n",
       "3           NaN      None                                               None   \n",
       "4           NaN      None                                               None   \n",
       "5           NaN       NaN                                                NaN   \n",
       "6           NaN      None                                               None   \n",
       "7           NaN       NaN                                                NaN   \n",
       "8     Reviewed;      None                                               None   \n",
       "9           NaN      None                                               None   \n",
       "10          NaN       NaN                                                NaN   \n",
       "11          NaN       NaN                                                NaN   \n",
       "12      256 AA.      None                                               None   \n",
       "\n",
       "0                                  DT                               DT  \\\n",
       "1    19-JUL-2004, sequence version 1.   02-JUN-2021, entry version 42.   \n",
       "2                                None                             None   \n",
       "3                                None                             None   \n",
       "4                                None                             None   \n",
       "5                                 NaN                              NaN   \n",
       "6                                None                             None   \n",
       "7                                 NaN                              NaN   \n",
       "8                                None                             None   \n",
       "9                                None                             None   \n",
       "10                                NaN                              NaN   \n",
       "11                                NaN                              NaN   \n",
       "12                               None                             None   \n",
       "\n",
       "0                                                  DE                   GN  \\\n",
       "1    RecName: Full=Putative transcription factor 0...   ORFNames=FV3-001R;   \n",
       "2                                                None                 None   \n",
       "3                                                None                 None   \n",
       "4                                                None                 None   \n",
       "5                                                 NaN                  NaN   \n",
       "6                                                None                 None   \n",
       "7                                                 NaN                  NaN   \n",
       "8                                                None                 None   \n",
       "9                                                None                 None   \n",
       "10                                                NaN                  NaN   \n",
       "11                                                NaN                  NaN   \n",
       "12                                               None                 None   \n",
       "\n",
       "0                                       OS  \\\n",
       "1    Frog virus 3 (isolate Goorha) (FV-3).   \n",
       "2                                     None   \n",
       "3                                     None   \n",
       "4                                     None   \n",
       "5                                      NaN   \n",
       "6                                     None   \n",
       "7                                      NaN   \n",
       "8                                     None   \n",
       "9                                     None   \n",
       "10                                     NaN   \n",
       "11                                     NaN   \n",
       "12                                    None   \n",
       "\n",
       "0                                                  OC  \\\n",
       "1    Viruses; Varidnaviria; Bamfordvirae; Nucleocy...   \n",
       "2                                                None   \n",
       "3                                                None   \n",
       "4                                                None   \n",
       "5                                                 NaN   \n",
       "6                                                None   \n",
       "7                                                 NaN   \n",
       "8                                                None   \n",
       "9                                                None   \n",
       "10                                                NaN   \n",
       "11                                                NaN   \n",
       "12                                               None   \n",
       "\n",
       "0                                                  OC  ...  \\\n",
       "1    Pimascovirales; Iridoviridae; Alphairidovirin...  ...   \n",
       "2                                                None  ...   \n",
       "3                                                None  ...   \n",
       "4                                                None  ...   \n",
       "5                                                 NaN  ...   \n",
       "6                                                None  ...   \n",
       "7                                                 NaN  ...   \n",
       "8                                                None  ...   \n",
       "9                                                None  ...   \n",
       "10                                                NaN  ...   \n",
       "11                                                NaN  ...   \n",
       "12                                               None  ...   \n",
       "\n",
       "0                                                  KW       FT  \\\n",
       "1    Activator; Reference proteome; Transcription;...    CHAIN   \n",
       "2                                                None      NaN   \n",
       "3                                                None      NaN   \n",
       "4                                                None      NaN   \n",
       "5                                                 NaN      NaN   \n",
       "6                                                None   1..256   \n",
       "7                                                 NaN      NaN   \n",
       "8                                                None     None   \n",
       "9                                                None     None   \n",
       "10                                                NaN      NaN   \n",
       "11                                                NaN      NaN   \n",
       "12                                               None     None   \n",
       "\n",
       "0                                            FT                     FT  \\\n",
       "1                                           NaN                    NaN   \n",
       "2                                           NaN                    NaN   \n",
       "3                                           NaN                    NaN   \n",
       "4                                           NaN                    NaN   \n",
       "5                                           NaN                    NaN   \n",
       "6                                           NaN                    NaN   \n",
       "7                                           NaN                    NaN   \n",
       "8                                           NaN                    NaN   \n",
       "9    /note=\"Putative transcription factor 001R\"   /id=\"PRO_0000410512\"   \n",
       "10                                          NaN                    NaN   \n",
       "11                                          NaN                    NaN   \n",
       "12                                         None                   None   \n",
       "\n",
       "0                        SQ  \\\n",
       "1                  SEQUENCE   \n",
       "2                   256 AA;   \n",
       "3                 29735 MW;   \n",
       "4   B4840739BF7D4121 CRC64;   \n",
       "5                       NaN   \n",
       "6                      None   \n",
       "7                       NaN   \n",
       "8                      None   \n",
       "9                      None   \n",
       "10                      NaN   \n",
       "11                      NaN   \n",
       "12                     None   \n",
       "\n",
       "0  MAFSAEDVLK EYDRRRRMEA LLLSLYYPND RKLLDYKEWS PPRVQVECPK APVEWNNPPS  \\\n",
       "1                                                None                  \n",
       "2                                                None                  \n",
       "3                                                None                  \n",
       "4                                                None                  \n",
       "5                                                 NaN                  \n",
       "6                                                None                  \n",
       "7                                                 NaN                  \n",
       "8                                                None                  \n",
       "9                                                None                  \n",
       "10                                                NaN                  \n",
       "11                                                NaN                  \n",
       "12                                               None                  \n",
       "\n",
       "0  EKGLIVGHFS GIKYKGEKAQ ASEVDVNKMC CWVSKFKDAM RRYQGIQTCK IPGKVLSDLD  \\\n",
       "1                                                None                  \n",
       "2                                                None                  \n",
       "3                                                None                  \n",
       "4                                                None                  \n",
       "5                                                 NaN                  \n",
       "6                                                None                  \n",
       "7                                                 NaN                  \n",
       "8                                                None                  \n",
       "9                                                None                  \n",
       "10                                                NaN                  \n",
       "11                                                NaN                  \n",
       "12                                               None                  \n",
       "\n",
       "0  AKIKAYNLTV EGVEGFVRYS RVTKQHVAAF LKELRHSKQY ENVNLIHYIL TDKRVDIQHL  \\\n",
       "1                                                None                  \n",
       "2                                                None                  \n",
       "3                                                None                  \n",
       "4                                                None                  \n",
       "5                                                 NaN                  \n",
       "6                                                None                  \n",
       "7                                                 NaN                  \n",
       "8                                                None                  \n",
       "9                                                None                  \n",
       "10                                                NaN                  \n",
       "11                                                NaN                  \n",
       "12                                               None                  \n",
       "\n",
       "0  EKDLVKDFKA LVESAHRMRQ GHMINVKYIL YQLLKKHGHG PDGPDILTVK TGSKGVLYDD  \\\n",
       "1                                                None                  \n",
       "2                                                None                  \n",
       "3                                                None                  \n",
       "4                                                None                  \n",
       "5                                                 NaN                  \n",
       "6                                                None                  \n",
       "7                                                 NaN                  \n",
       "8                                                None                  \n",
       "9                                                None                  \n",
       "10                                                NaN                  \n",
       "11                                                NaN                  \n",
       "12                                               None                  \n",
       "\n",
       "0  SFRKIYTDLG WKFTPL  \n",
       "1               None  \n",
       "2               None  \n",
       "3               None  \n",
       "4               None  \n",
       "5                NaN  \n",
       "6               None  \n",
       "7                NaN  \n",
       "8               None  \n",
       "9               None  \n",
       "10               NaN  \n",
       "11               NaN  \n",
       "12              None  \n",
       "\n",
       "[12 rows x 47 columns]"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7e916e94",
   "metadata": {},
   "outputs": [],
   "source": [
    "df.to_csv('test.xls')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "aac1bc77",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "547832d1",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
